astropy/io/fits/tests/test_connect.py

```diff
@@ -329,3 +329,13 @@ def test_unit_warnings_read_write(tmpdir):
     with catch_warnings() as l:
         Table.read(filename, hdu=1)
     assert len(l) == 0
+
+
+def test_convert_comment_convention(tmpdir):
+    """
+    Regression test for https://github.com/astropy/astropy/issues/6079
+    """
+    filename = DATA + '/table_with_comment.fits'
```

Comment:
Do we really need a whole new data file just for this test? I think we can just add a COMMENT to an existing file like `data/table.fits`?

Also, use `os.path.join`, not `/` to ensure Windows can read this.